data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,128 @@ mission "Strider notification"
+			choice
+				`	"What makes you think I'm parking my ship at a haunted station?"`
+					goto ghosts
+				`	"Alright then, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			label ghosts
+				`	Likeet gives a laugh. "It's not actually haunted, of course. Those are just stories. Do you think I'd send an entire film crew and all their expensive equipment to a place that was actually dangerous?"`
+			choice
+				`	"Alright then, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+						decline
+			label accept
+				`	"Excellent! I'll let the crew know where to find you."`
+					accept
+	on visit
+		dialog `You've reached <planet>, but your cargo or passengers are on another ship! Better take off and wait for them to arrive in this system.`
+	on complete
+		event "film crew" 30
+		payment 100000 50
+		conversation
+			`You help your passengers unload all of their film equipment once you arrive on <planet>. The props seem to get more and more elaborate as they are unpacked from your ship. You note the presence of countless rubber limbs, mechanical monsters, and cans of fake blood being carried onto the set. Once you are finished, Likeet approaches you.`
+			choice
+				`	"What's the name of your movie, anyways?"`
+				`	"What's your movie about, anyways?"`
+					goto movie
+				`	"I better get going, then."`
+					goto going
+			label movie
+			`	"I don't want to spoil anything, but it's a psychological horror film that explores the nature of human isolation, and the shadow that our own mortality casts over our lives." She continues to elaborate on more philosophical themes that seemingly have less and less to do with the actual film, until she notices that your eyes have long since glazed over.`
+			`	"Well, anyways, thanks for your help, captain. I'll contact you in a month once we're finished," says Likeet.`
+					accept
+			label going
+			`	"Thanks for your help, captain. I'll contact you in a month once we're finished," says Likeet.`
+					accept
+
+
+
+event "film crew"
+
+
+
+mission "Hai Film Crew [2]"
+	name "Hai Film Crew"
+	description "Go to <destination> to pick up the film crew you transported there."
+	destination "Darkrest"
+	to offer
+		has "event: film crew"
+	on offer
+		dialog
+			`Upon landing your ship, you are startled by a vaguely familiar voice coming from your intercom. You soon recognize it as Likeet, the movie director that you brought to Darkrest a month ago.`
+			`	"Captain <last>! Good to see you again! We've finished filming here and need a ride off this place. I'll see you soon." Before you can say anything, she hangs up.`
+				accept
+
+
+
+mission "Hai Film Crew [3]"
+	name "Hai Film Crew"
+	description "Bring the movie crew you transported from <source> to their studio on <destination>."
```

Comment:
```suggestion
	description "Take Likeet's film crew home to their studio on <destination>. Payment is <payment>."
```
Since `<source>` is undefined as far as I can tell, and `<origin>` would return Darkrest.
You might also want to add more detail here.